examples/create-setters-simple/README.md

```diff
@@ -46,50 +24,29 @@ kind: ConfigMap
 metadata:
   name: create-setters-fn-config
 data:
-  app: nginx
-  image: ubuntu
-  role: |
+  nginx-replicas: "4"
+  env: |
     - dev
-    - pro
-  tag: 1.1.2
-```
-
-Invoking `create-setters` function would add the setter comments.
-
-```yaml
-# resources.yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: ubuntu-development # kpt-set: ${image}-development
-data:
-  some-key: some-value
----
-apiVersion: v1
-kind: MyKind
-metadata:
-  name: ubuntu # kpt-set: ${image}
-image: nginx:1.1.2 # kpt-set: ${app}:${tag}
-roles: # kpt-set: ${role}
-  - dev
-  - pro
+    - stage
+  tag: 1.16.1
 ```
 
 ### Function invocation
 
-Invoke the function by running the following commands:
+Invoke the function by running the following command:
 
 ```shell
 $ kpt fn render create-setters-simple
 ```
 
 ### Expected result
 
-`Comment` is added to the resource with the `Value` given below as they match the `Setters`.
+`Setter comment` is added to the resource with the `Value` given below as they match the `Setter`.
```

Comment:
not sure you want to backtick `Setter comment`.

Reply:
Removed the table and added sentences to explain better.